modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,406 @@
+new-topic-name-value   1         1    PROTOBUF
+----
+
+The Iceberg table is inside a namespace called `redpanda`, and has the same name as the Redpanda topic name. As you produce records to the topic, the data also becomes available in object storage for consumption by Iceberg-compatible clients. 
+
+== Schema support and mapping
+
+The `redpanda.iceberg.mode` property determines how Redpanda maps the topic data to the Iceberg table structure. You can either have the generated Iceberg table match the stucture of a Avro or Protobuf schema in the Schema Registry, or use the schemaless mode where Redpanda stores the record values as-is in the table. 
+
+The JSON Schema format is not supported in this beta release. If your topic data is in JSON, it is recommended to use the `key_value` mode.
+
+=== Iceberg modes and table schemas
+
+For both `key_value` and `value_schema_id_prefix` modes, Redpanda writes to a `redpanda` table column that stores a single Iceberg struct per record, containing nested columns of the metadata from each record, including the record key, headers, timestamp, the partition it belongs to, and its offset. 
+
+For example, if you produce to a topic according to the following Avro schema:
+
+[,avro]
+----
+{
+    "type": "record",
+    "name": "ClickEvent",
+    "fields": [
+        {
+            "name": "user_id",
+            "type": "int"
+        },
+        {
+            "name": "event_type",
+            "type": "string"
+        },
+        {
+            "name": "ts",
+            "type": "string"
+        }
+    ]
+}
+----
+
+The `key_value` mode writes to the following table format:
+
+[,sql]
+----
+CREATE TABLE ClickEvent ( 
+    redpanda OBJECT( 
+        partition INT, 
+        timestamp TIMESTAMP, 
+        offset LONG, 
+        headers ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)), 
+        key BINARY NULLABLE
+    ), 
+    value BINARY NULLABLE
+)
+----
+
+Consider this schemaless approach if the topic data is in JSON, or if you can use the Iceberg data in its semi-structured format.
+
+The `value_schema_id_prefix` mode translates to the following table format:
+
+[,sql]
+----
+CREATE TABLE ClickEvent (
+    redpanda OBJECT(
+        partition INT,
+        timestamp TIMESTAMP,
+        offset    LONG,
+        headers   ARRAY(OBJECT(KEY BINARY, VALUE BINARY NULLABLE)),
+        key       BINARY NULLABLE
+    ),
+    user_id INT,
+    event_type STRING NULLABLE,
+    ts STRING NULLABLE
+)
+----
+
+With schema integration, Redpanda uses the schema ID prefix embedded in each record to find the matching schema in the Schema Registry. Producers to the topic must use the schema ID prefix in the serialization process so Redpanda can determine the schema used for each record, parse the record according to that schema, and use the schema for the Iceberg table as well.
+
+=== Schema types translation
+
+Redpanda supports direct translations of the following types to Iceberg value domains:
+
+[tabs]
```

Comment:
@rockwotj is this closer to what you had in mind? Also, please double check the tables -- especially Avro `fixed` which I couldn't quite figure out what the Iceberg equivalent would be